smithy-model/src/test/resources/software/amazon/smithy/model/selector/resource-tagging.smithy

```diff
@@ -0,0 +1,47 @@
+$version: "2.0"
```

Comment:
What's resource tagging to do with the property selector test?

Reply:
Trimmed